app/controllers/prior_authority/steps/further_information_controller.rb

```diff
@@ -0,0 +1,88 @@
+module PriorAuthority
+  module Steps
+    class FurtherInformationController < BaseController
+      skip_before_action :verify_authenticity_token, only: [:create, :destroy]
+
+      def edit
+        @form_object = FurtherInformationForm.build(record, application: current_application)
+        @explanation = record.information_requested
+        @supporting_documents = record.supporting_documents
+      end
+
+      def create
```

Comment:
nit: A loooot of this class is a copy paste from another controller. It'd be nice to maybe move the shared stuff into a shared `MultiFileUploadable` concern or something.